css/index.css

```diff
@@ -1,3 +1,281 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container{
+	width: 900px;
+	margin: 0 auto;
+}
+
+header{
+	border-bottom: 1px solid grey;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 20px;
+	width: 100%;
+}
+
+nav{
+	width: 80%;
+}
+
+nav ul{
+	display: flex;
+	justify-content: space-evenly;
+}
+
+
+nav ul li{
+	display: inline;
+}
+
+nav ul li a{
+	color: grey;
+	text-decoration: none;
+}
+
+@media (max-width:800px){
+	.container{
+		width: 100%;
+		margin: 0 auto;
+	}
+	header{
+		display: flex;
+		flex-direction: column-reverse;
+		padding: 20px;
+		border-bottom: 1px solid white;
+	}
+
+	.header-image{
+		margin-bottom: 20px;
+	}
+
+	nav{
+		width: 100%;
+	}
+
+	nav ul{
+		width: 100%;
+	}
+
+}
+
+@media (max-width:500px){
+	.container{
+		width: 100%;
+		margin: 0 auto;
+	}
+	header{
+		display: flex;
+		flex-direction: column-reverse;
+		border-bottom: 1px solid white;
+		padding-bottom: 0;
+	}
+
+	.header-image{
+		border-bottom: 1px solid black;
+		width: 100%;
+		display: flex;
+		justify-content: center;
+		margin-bottom: 0;
+		padding-bottom: 20px;
+		margin-bottom: 10px;
+	}
+
+	nav{
+		width: 100%;
+		margin: 0 auto;
+	}
+
+	nav ul{
+		width: 100%;
+		display: block;
+	}
+
+
+	nav ul li{
+		display: block;
+		border-bottom: 1px solid black;
+		width: 100%;
+		text-align: center;
+		padding-bottom: 20px
```

Comment:
You could probably add a padding top here to give space to your nav li items in mobile view and stop them from hugging the top-border